.github/workflows/precommit.yml

```diff
@@ -31,10 +33,10 @@ jobs:
       - name: Install mypy
         run: |
           set -x
-          pip install mypy types-requests types-jmespath pandas-stubs
+          pip install -r requirements-dev.txt
```

Comment:
i think adding the dependencies to the pyproject.yaml (poetry add --dev ...) would be more consistent with the current setup, right?

Reply:
yes. old habits...
fixed.